src/Template/Element/plugin_class_error.ctp

```diff
@@ -29,5 +29,3 @@ else:
         '<a href="http://book.cakephp.org/3.0/en/plugins.html#autoloading-plugin-classes">Plugins - autoloading plugin classes</a>'
     );
 endif;
-
-?>
```

Comment:
It makes it easier to add HTML below - without the hasle of having to add the closing tag again as well.